src/workerfarm/child.js

```diff
@@ -10,23 +14,19 @@ class Child {
     this.responseId = 0;
     this.maxConcurrentCalls = 10;
 
-    process.env.PARCEL_WORKER_TYPE = 'remote-worker';
+    // Assigning parcelWorker variables to process
+    // Slightly improves the usability of bi-directional request
+    // in places that don't have access to child or workerfarm
+    // in a reliable way (for example the tests of Parcel)
+    process.parcelWorker = true;
+    process.parcelRequest = this.addCall.bind(this);
```

Comment:
looks like this is only used in the tests now? can the tests also just call `WorkerFarm.callMaster`?

Reply:
We'll have to handle node 6 and 8 but sure, I'll change. I'd like to keep `process.parcelWorker = true;`, unless you prefer the env variable as I mentioned earlier